sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1390,6 +1539,14 @@ std::vector<node> get_root_nodes() const;
 ----
 |Returns a list of all nodes in the graph which have no dependencies.
 
+|
+[source,c++]
+----
+size_t get_mem_use_current() const;
+----
+|Returns the size in bytes of the physical memory currently being used by this
+graph for <<graph-memory-allocations, graph-owned memory allocations>>.
```

Comment:
Would say that "current" is implicit and we could shorted this to `get_mem_use".

Also thinking if "used" is too generic a description,  as it's scope could be interpreted as larger than just the memory allocated by the graph. `get_mem_alloc_size()` or something that references allocation explicitly might be better.

Reply:
Works for me, I've updated this to `get_mem_alloc_size()` now and adjusted the description to reflection that.